data/wordlists.yml

```diff
@@ -10550,13 +10550,13 @@ uplay:
   categories:
   - misc
 
-13:
+thirteen:
   url: https://download.weakpass.com/wordlists/768/13.dic.gz
   summary: Misc wordlist
   categories:
   - misc
 
-02:
+two:
```

Comment:
Too generic. Needs to indicate where the wordlist is from and what does it contain.